### ISSUE TITLE
MetalLB lab uses an element from the "Exposing apps using a Route" lab

### ISSUE THREAD

**Issue by augustrh** [rhpds/roadshow_ocpvirt_instructions#32]
TL;DR: we should rearrange the navigational list to have attendees run the Route lab before the MetalLB lab as the latter uses an outcome of the former.

In step 3 of the metalLB lab, "Expose the database node externally" it uses a database exposed by a route in the lab that's listed after it in the navigation al menu ("Exposing apps using a Route"). 

While the labs can be done in any order if there's a dependency on one we should probably both mention that in the lab's intro and list them in the order they need to be run as it's common for people to follow the navigational list from top to bottom.

**Comment by joelapatatechaude** [comment]
100% agree, also got confused myself on my dry run. I created a PR @augustrh feel free to review / comment / advocate for :)
https://github.com/rhpds/roadshow_ocpvirt_instructions/pull/45
